### ISSUE TITLE
Useless use of echo.

### ISSUE THREAD

**Issue by luis-lavaire** [6gk/fet.sh#6]
#### TL;DR:
`set -- $(echo ...)` is useless use of `echo`.

---

It happens three times:

- https://github.com/6gk/fet.sh/blob/master/fet.sh#L123: `set -- $(echo /var/lib/pacman/local/*)`
- https://github.com/6gk/fet.sh/blob/master/fet.sh#L128: `set -- $(echo /var/db/xbps/.*)`
- https://github.com/6gk/fet.sh/blob/master/fet.sh#L133: `set -- $(echo /var/db/pkg/*/*)`

There's no need of `$(echo ...)`; the shell will expand the globbing patterns, and needs no "help".

Also, instead of three `pkgs=$#`'s, use only one after performing the checks (i.e., after [line 136](https://github.com/6gk/fet.sh/blob/master/fet.sh#L136)).

**Comment by 6gk** [comment]
@vizs what have you say about your bad code.

jk.

Thank you, I'll fix it.

**Comment by luis-lavaire** [comment]
You're welcome. I love the idea of this script.

**Comment by vizs** [comment]
This is what happens when you shell script right after you wake up I guess :P
